### PR TITLE
Mj us moment ticker and copy

### PR DESCRIPTION
**Pull request by michaelbjacobson**
## What does this PR do?
Updates campaign from Aus moment to US end of year appeal, including copy and ticker styling.

## Screenshot
Screenshot 2021-11-22 at 10 51 47

### CODE REVIEW

File: support-frontend/assets/components/ticker/contributionTicker.scss [hunk 1]
```diff
@@ -59,9 +59,7 @@ $progress-bar-height: 10px;
 	}
 }
 
-.contributions-landing-ticker__label {
-	font-style: italic;
-}
+.contributions-landing-ticker__label {}
```
Comment (marialani) [inline support-frontend/assets/components/ticker/contributionTicker.scss:62]: Can we get rid of this?

File: support-frontend/assets/helpers/campaigns/campaigns.ts [hunk 2]
```diff
@@ -2,81 +2,84 @@ import type { TickerSettings } from 'components/ticker/contributionTicker';
...
 
 export function getCampaignSettings(
-	campaignCode?: string,
+  campaignCode?: string,
 ): CampaignSettings | null {
-	if (campaignEnabledForUser(campaignCode)) {
-		return campaign;
-	}
+  if (campaignEnabledForUser(campaignCode)) {
+    console.log('campaignEnabledForUser: ', campaignEnabledForUser(campaignCode))
```
Comment (marialani) [inline support-frontend/assets/helpers/campaigns/campaigns.ts:70]: Sneaky console log, one slipped by me a little while back, am thinking we could add in a pre-commit check

**Comment by marialani** [comment]
Are we able to see a preview of what the moment would look like?

File: support-frontend/assets/components/ticker/contributionTicker.scss (hunk 1 above)
Comment (michaelbjacobson) [reply support-frontend/assets/components/ticker/contributionTicker.scss:62]: We sure can :)

File: support-frontend/assets/helpers/campaigns/campaigns.ts (hunk 2 above)
Comment (michaelbjacobson) [reply support-frontend/assets/helpers/campaigns/campaigns.ts]: Thanks @marialani!

**Review by marialani — DISMISSED** [review]
👍

File: support-frontend/assets/helpers/campaigns/campaigns.ts [hunk 3]
```diff
@@ -2,81 +2,82 @@ import type { TickerSettings } from 'components/ticker/contributionTicker';
 import type { ContributionTypes } from 'helpers/contributions';
 
 type CampaignCopy = {
-	headerCopy?: string | JSX.Element;
-	contributeCopy?: string | JSX.Element;
+  headerCopy?: string | JSX.Element;
+  contributeCopy?: string | JSX.Element;
```
Comment (TomPretty) [inline support-frontend/assets/helpers/campaigns/campaigns.ts:6]: Looks like we've maybe reformatted tabs -> spaces here? The linting currently wants tabs so probably worth swapping them back

File: support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.tsx [hunk 4]
```diff
@@ -250,8 +248,7 @@ function withProps(props: PropTypes) {
 				>
 					<SecureTransactionIndicator modifierClasses={['top']} />
 
-					{props.canShowTicker &&
-					campaignSettings &&
+					{campaignSettings &&
```
Comment (TomPretty) [inline support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.tsx:251]: I think we can wap this for `campaignSettings?.tickerSettings ?`

File: support-frontend/typescript-errors.json [hunk 5]
```diff
@@ -732,6 +731,10 @@
             "path": "assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx",
             "message": "Parameter 'result' implicitly has an 'any' type."
         },
+        {
+            "path": "assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx",
+            "message": "Parameter 'result' implicitly has an 'any' type."
+        },
```
Comment (TomPretty) [inline support-frontend/typescript-errors.json:737]: I think maybe running `yarn` to install latest deps will fix this one

**Comment by michaelbjacobson** [comment]
> Are we able to see a preview of what the moment would look like?

I've added a screenshot to the PR now :)

File: yarn.lock [hunk 6]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (TomPretty) [inline yarn.lock:4]: we can get rid of this one!

File: support-frontend/typescript-errors.json (hunk 5 above)
Comment (TomPretty) [reply support-frontend/typescript-errors.json:737]: I think you'll just need to run yarn in the `support-frontend` sub dir & then `yarn ts-errors` to regenerate this file

**Comment by marialani** [comment]
> > Are we able to see a preview of what the moment would look like?
> 
> I've added a screenshot to the PR now :)

I think the so far count and goal reached are the same size this time around. They should both be 17px on desktop and 15px on mobile.

File: yarn.lock (hunk 6 above)
Comment (michaelbjacobson) [reply yarn.lock]: Good shout, I must've generated it by mistake.

File: support-frontend/typescript-errors.json (hunk 5 above)
Comment (michaelbjacobson) [reply support-frontend/typescript-errors.json:737]: Done :)

**Review by TomPretty — APPROVED** [review]
Great stuff! 🚀

**Pull request merged**